examples/5.Papa-Iridium-Example/Papa-DishDuck/Papa-DishDuck.ino

```diff
@@ -100,6 +100,15 @@ std::string toTopicString(byte topic) {
     case topics::health:
       topicString ="health";
       break;
+    case topics::bmp180:
+      topicString ="bmp";
```

Comment:
Since we have two different types of BMPs it might be better to be more descriptive than "bmp". Or just combine them into one